ncvalidator causes -ansi build to fail with some minor warnings

In today's master, the -ansi build fails:

```
../liblib/libnetcdf.la -lhdf5_hl -lhdf5 -lm -ldl -lz -lcurl 
/bin/bash ../libtool  --tag=CC   --mode=link gcc  -Wall -g -ansi  -L/usr/local/zlib-1.2.11/lib -L/usr/local/szip-2.1/lib -L/usr/local/hdf5-1.10.4/lib -o nccopy nccopy.o nciter.o chunkspec.o utils.o dimmap.o list.o ../liblib/libnetcdf.la -lhdf5_hl -lhdf5 -lm -ldl -lz -lcurl 
ncvalidator.c: In function ‘check_little_endian’:
ncvalidator.c:281:5: error: C++ style comments are not allowed in ISO C90
     // return 0 for big endian, 1 for little endian.
     ^
ncvalidator.c:281:5: error: (this will be reported only once per input file)
ncvalidator.c: In function ‘main’:
ncvalidator.c:2291:17: warning: implicit declaration of function ‘getopt’; did you mean ‘getgid’? [-Wimplicit-function-declaration]
     while ((i = getopt(argc, argv, "xthq")) != EOF)
                 ^~~~~~
                 getgid
Makefile:1215: recipe for target 'ncvalidator.o' failed
make[2]: *** [ncvalidator.o] Error 1
```

--- a/ncdump/ncvalidator.c
+++ b/ncdump/ncvalidator.c
@@ -278,7 +278,7 @@ static const char ncmagic[] = {'C', 'D', 'F', 0x01};
 
 static int check_little_endian(void)
 {
-    // return 0 for big endian, 1 for little endian.
+    /* return 0 for big endian, 1 for little endian. */
     volatile uint32_t i=0x01234567;
     return (*((uint8_t*)(&i))) == 0x67;
 }
@@ -1104,8 +1104,8 @@ hdr_get_name(int          fd,
             /* This is considered not a fatal error, we continue to validate */
             if (verbose) printf("Error @ [0x%8.8zx]:\n", err_addr);
             if (verbose) printf("\t%s \"%s\": name padding is non-null byte\n", loc, *namep);
-            // free(*namep);
-            // *namep = NULL;
+            /* free(*namep);
+               *namep = NULL; */
             DEBUG_ASSIGN_ERROR(err, NC_ENULLPAD)
             if (repair) {
                 val_repair(fd, err_addr, (size_t)padding, (void*)nada);

